### DISCUSSION TITLE
Welcome to OpenFPGA Discussions!

### DISCUSSION THREAD

**Discussion by tangxifan** [lnis-uofu/OpenFPGA#376] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ganeshgore** [comment]
This is great

**Comment by GSBhatta** [comment]
How does OpenFPGA generate SDC files? Based on what technology are the clock frequency along with delays set? To which technology node is the Standard cell mapped to?

**Reply by tangxifan** [reply]
@Gurusatwik  
- You may refer to the https://openfpga.readthedocs.io/en/master/manual/openfpga_shell/openfpga_commands/fpga_sdc_commands/ for SDC generation. Note that SDC files are highly related to your case. So, there are plenty of options for your actual needs. 
- You can provide the clock frequency through the simulation setting file https://openfpga.readthedocs.io/en/master/manual/arch_lang/simulation_setting/ SDC generator will honor it.
- There is no restrictions on technology. You define what you want. We have silicon proofs on TSMC 22nm, Skywater 180nm etc. Researchers have tried a wide range of technologies, from 180nm down to 12nm. No issues shown.

**Reply by GSBhatta** [reply]
- When the operating frequency is set to "auto", how will the VPR STA Engine analyze the maximum operating frequency of the fabric? It must need a standard cell library to analyze if I am not wrong.
- Isn't there any standard cell library in OpenFPGA to which the Fabric netlists are mapped to?
-  What is the theory behind the frequency value set for clk[0] as shown in the below image?
<img width="493" alt="image" src="https://github.com/lnis-uofu/OpenFPGA/assets/113631826/faf4899f-7784-4d9b-9677-28566188172d">
